drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -28,6 +28,14 @@ def self.configuration
       Hmis::Hud::Enrollment.hmis_configuration(version: '2024')
     end
 
+    # check for the most minimal permission needed to resolve this object
+    def self.authorized?(object, ctx)
+      permissions = [:can_view_enrollment_details, :can_view_limited_enrollment_details]
+      super && permissions.any? do |permission|
+        GraphqlPermissionChecker.current_permission_for_context?(ctx, permission: permission, entity: object)
+      end
```

Comment:
Technically you need `(can_view_project AND can_view_enrollment_details) OR can_view_limited_enrollment_details` so we should check all that here if we can :\

Reply:
Okay, I think I've represented that auth logic